README.md

```diff
@@ -97,10 +97,18 @@ In **PowerShell** or in a **Command Prompt** window:
     ```
     Run `brew info` to get the latest environment variable export suggestions, such as `brew info zlib`
 
+    Install/Upgrade **Xcode** in case you still find problems with those libraries.
+
+  In some cases, the Python installer have conflicts with the Python version installed by default on macOS. To prevent this problem, install pyenv:
```

Comment:
As discussed in [this comment](https://github.com/aws/aws-elastic-beanstalk-cli-setup/pull/64#issuecomment-561906697), we don't want this.